gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8-bin.zip
```

Comment:
It would be awesome if you could update the version specified under the Wrapper task as well: https://github.com/jaegertracing/jaeger-client-java/blob/master/build.gradle#L40